### DISCUSSION TITLE
complicated.py Exception AttributeError module 'pyqtgraph.Qt.QtGui' has no attribute 'QApplication'

### DISCUSSION THREAD

**Discussion by alexlond** [highfestiva/finplot#384] [Q&A] [answered]
Noob here!
Do you know how to solve the following error in the example file https://github.com/highfestiva/finplot/blob/master/finplot/examples/complicated.py:

Exception has occurred: AttributeError
module 'pyqtgraph.Qt.QtGui' has no attribute 'QApplication'
  File "C:\Users\al\Documents\crypto\complicated.py", line 436, in <module>
    fplt.show()

My python version is 3.9.7 (default, Sep 16 2021, 16:59:28) [MSC v.1916 64 bit (AMD64)]
These are the versions of my modules:
finplot==1.8.4
PyQt5==5.15.7
PyQt5-Qt5==5.15.2
PyQt5-sip==12.11.0
pyqtgraph==0.13.1

aiohttp==3.8.1
aiosignal==1.2.0
async-timeout==4.0.2
attrs==21.4.0
certifi==2022.6.15
charset-normalizer==2.1.0
cycler==0.11.0
dateparser==1.1.1
fonttools==4.34.4
frozenlist==1.3.0
idna==3.3
kiwisolver==1.4.4
matplotlib==3.5.2
multidict==6.0.2
numpy==1.23.1
packaging==21.3
pandas==1.4.3
Pillow==9.2.0
pyparsing==3.0.9
python-binance==1.0.16
python-dateutil==2.8.2
pytz==2022.1
pytz-deprecation-shim==0.1.0.post0
regex==2022.3.2
requests==2.28.1
six==1.16.0
tzdata==2022.1
tzlocal==4.2
ujson==5.4.0
urllib3==1.26.10
websocket-client==1.4.2
websockets==10.3
yarl==1.7.2

**Comment by highfestiva** [comment, marked as answer]
I unfortunately think that finplot is incompatible with the newer versions of PyQt. Try installing an older version, to see if that helps. I'll try getting around to fixing this some day soon... :-/
